### PR TITLE
Replace LS page with CAPP Agile Learning page (NL)

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This PR aims on a new landing page to replace LearningSpaces branding with **CAPP Agile Learning** branding on the Dutch website, as is the plan to do before NextLearning April 18th, 2017.

# Done
- [x] Add basic page
- [x] Redirect learningspaces.html to capp-agile-learning.html on NL locale
- [x] Change all internal learningspaces links to capp-agile-learning
- [x] Mobile 
- [x] Client logos
- [x] Hero image
- [x] Logo on product overview
- [x] Icons
- [x] Screenshots
- [x] Content @mildaskl 
- [x] Branding @sn3p 
- [x] Demo image

# Todo

- [ ] Review

# Screenshot

screenshot-capp-agile-learning-v1

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -337,8 +338,18 @@ def root_url?
...
   def logo
-    image_link = root_url? ? "logos/defacto-invert" : "logos/defacto"
+    if root_url?
+      image_link = "logos/defacto-invert"
+    elsif capp_agile_url?
+      image_link = "logos/defacto-white"
+    else
+      image_link = "logos/defacto"
+    end
     img = image_tag("#{image_link}.svg", alt: "Defacto", onerror: "this.src='/images/#{image_link}.png'; this.onerror=null;")
```
Comment (fatboypunk) [inline config.rb]: This could remove the last else right?
```ruby
image_tag("logos/defacto#{"image_link}.svg" ....)
```

**Comment by fatboypunk** [comment]
Be aware of the empty white spots 
screen shot 2017-03-16 at 11 57 59

**Comment by fatboypunk** [comment]
Are there plans to also do this for the german pages in the future?

File: source/localizable/_hero_home.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero-inner">
+    <div class="hero-copy">
+      <h1><%= t("home.hero.title") %></h1>
+      <h2><%= t("home.hero.text") %></h2>
+      <%= nav_link_to t("actions.home.demo"), "/demo-aanvragen.html", "data-event": "home_demo", class: "button" %>
+      <%= nav_link_to t("actions.home.productsheet"), "/productsheet-downloaden.html", "data-event": "home_produktblatt", class: "button white outline" %>
```
Comment (sn3p) [inline source/localizable/_hero_home.erb]: Should `"data-event": "home_produktblatt` be German?

File: source/localizable/capp-agile-learning.nl.erb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: CAPP Agile Learning
+description:
+  nl: Performance support voor elke organisatie. Vraag nu een demo aan.
+  de: meta
```
Comment (sn3p) [inline source/localizable/capp-agile-learning.nl.erb]: Remove this until we need it?

File: source/localizable/capp-agile/_capp-agile-testimonials.erb [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<%
+all_testimonials = data["learningspaces-testimonials_" + I18n.locale.to_s]
+testimonials = all_testimonials.clone
+%>
+
+<section class="testimonials gray cols">
+
+  <div class="row">
+
+  <h2><%= t("capp-agile.testimonials_heading") %></h2>
```
Comment (sn3p) [inline source/localizable/capp-agile/_capp-agile-testimonials.erb]: Indentation 👮

File: source/localizable/capp-agile-learning.nl.erb (hunk 3 above)
Comment (RonaldDijkstra) [inline source/localizable/capp-agile-learning.nl.erb]: Yes

File: source/localizable/capp-agile/_capp-agile-testimonials.erb (hunk 4 above)
Comment (RonaldDijkstra) [inline source/localizable/capp-agile/_capp-agile-testimonials.erb]: Yes

**Comment by sn3p** [comment]
Minor but reporting anyway. Logo jumps when mobile menu opens:

screen shot 2017-03-22 at 17 16 14

**Comment by RonaldDijkstra** [comment]
Here #502

**Comment by sn3p** [comment]
The side-by-side images and texts could use some mobile enhancement.
An improvement would be ordering it like this on mobile:

```
[image]
[text]

[image]
[text]
```

... which can be accomplished by using flex [`order`](https://developer.mozilla.org/en/docs/Web/CSS/order).

**Comment by sn3p** [comment]
Good to go! Props 👍

**Pull request merged**